routes/web.php

```diff
@@ -32,3 +32,8 @@
 Route::resource('lessons', 'LessonController');
 
 Route::resource('quiz', 'QuizController');
+
+Route::resource('answer', 'AnswerController');
+
+Route::post('/completed-quiz/{quiz}', 'CompletedQuizController@store');
```

Comment:
use `group` function for routes to have more organize structure

Reply:
I realize that this route is unused in the future. I have now removed it.